### PR TITLE
🔧 footer hello links

### LINKED ISSUE #5900: Refresh links in Footer with new Hello urls
we will be removing 'knowledge-base' from url and polishing URLS 

When this would be closed we can proceed
- https://github.com/kodadot/ops-internal/issues/396

### PR DESCRIPTION
**Pull request by roiLeo**
First PR on M2 Chip

## PR Type

- [x] Bugfix

## Context

- [x] Closes #5900

### CODE REVIEW

File: components/TheFooter.vue [hunk 1]
```diff
@@ -142,7 +142,7 @@ const { isDarkMode } = useTheme()
 const menuMarketplace: Menu[] = [
   {
     name: $i18n.t('faq.page'),
-    url: 'https://hello.kodadot.xyz/knowledgebase/general-1/faq',
+    url: 'https://hello.kodadot.xyz/general-1/faq',
```
Comment (yangwao) [inline components/TheFooter.vue:145]: https://hello.kodadot.xyz/ecosystem/faq

File: components/TheFooter.vue [hunk 2]
```diff
@@ -159,7 +159,7 @@ const menuMarketplace: Menu[] = [
 const menuKodadot: Menu[] = [
   {
     name: $i18n.t('about'),
-    url: 'https://hello.kodadot.xyz/knowledgebase/general/faq',
+    url: 'https://hello.kodadot.xyz/general/faq',
```
Comment (yangwao) [inline components/TheFooter.vue:162]: https://hello.kodadot.xyz/about-us/who-are-we

File: components/TheFooter.vue [hunk 3]
```diff
@@ -174,7 +174,7 @@ const menuKodadot: Menu[] = [
 
   {
     name: $i18n.t('referralProgram'),
-    url: 'https://hello.kodadot.xyz/knowledgebase/technical/kodadots-programs/artist-referral-program',
+    url: 'https://hello.kodadot.xyz/technical/kodadots-programs/artist-referral-program',
```
Comment (yangwao) [inline components/TheFooter.vue:177]: https://hello.kodadot.xyz/be-part-of-kodadot/kodadots-programs/artist-referral-program

File: components/TheFooter.vue [hunk 4]
```diff
@@ -185,13 +185,13 @@ const menuKodadot: Menu[] = [
 
   {
     name: $i18n.t('artist ambassador'),
-    url: 'https://hello.kodadot.xyz/knowledgebase/technical/kodadots-programs/artist-ambassador-program',
+    url: 'https://hello.kodadot.xyz/technical/kodadots-programs/artist-ambassador-program',
```
Comment (yangwao) [inline components/TheFooter.vue:188]: https://hello.kodadot.xyz/be-part-of-kodadot/kodadots-programs/artist-ambassador-program

File: components/TheFooter.vue [hunk 5]
```diff
@@ -185,13 +185,13 @@ const menuKodadot: Menu[] = [
...
     external: true,
   },
 
   {
     name: $i18n.t('ambassador program'),
-    url: 'https://hello.kodadot.xyz/knowledgebase/technical/kodadots-programs/ambassador-program',
+    url: 'https://hello.kodadot.xyz/technical/kodadots-programs/ambassador-program',
```
Comment (yangwao) [inline components/TheFooter.vue:194]: https://hello.kodadot.xyz/be-part-of-kodadot/kodadots-programs/ambassador-program

**Review by yangwao — CHANGES_REQUESTED** [review]
@JustLuuuu supposed post new updated link, there was following changes

**Review by preschian — APPROVED** [review]

**Review by daiagi — APPROVED** [review]

**Comment by yangwao** [comment]
mmm

image

I would also 
- deleted completely Contribute page from this codebase (as we want maintain less same files)
- link Contribute to https://hello.kodadot.xyz/be-part-of-kodadot/join-as-a-developer

**Comment by roiLeo** [comment]
> * https://hello.kodadot.xyz/be-part-of-kodadot/join-as-a-developer

Done, do we keep `CONTRIBUTING.md`?

**Comment by JustLuuuu** [comment]
I would also love to have tutorials in the footer:
https://hello.kodadot.xyz/tutorial/

**Comment by roiLeo** [comment]
> @JustLuuuu is everything oki?

Nope wait for last commit

**Comment by roiLeo** [comment]
😢 
Capture d’écran 2023-05-12 à 2 14 51 PM

**Pull request closed without merging**